recipes/default.rb

```diff
@@ -63,3 +63,17 @@
   end
   action %i[create enable start]
 end
+
+# Bash completion
+package 'bash-completion'
+
+uri_list = %w[gobgp-completion.bash gobgp-dynamic-completion.bash gobgp-static-completion.bash]
+url = "https://raw.githubusercontent.com/osrg/gobgp/v#{node['gobgp']['binary']['version']}/tools/completion/"
+dest = '/etc/bash_completion.d/'
+
+uri_list.each do |uri|
+  remote_file uri do
```

Comment:
Could you add a way to optionally add the file checksum?
Otherwise you are going to have issues with our internal "remote_file" mechanism